metrics/metrics.go

```diff
@@ -30,21 +30,49 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
-// APIRequests is a counter vector for requests to endpoints
-var APIRequests = promauto.NewCounterVec(prometheus.CounterOpts{
-	Name: "api_endpoints_requests",
-	Help: "The total number of requests per endpoint",
-}, []string{"endpoint"})
-
-// APIResponsesTime collects the information about api response time per endpoint
-var APIResponsesTime = promauto.NewHistogramVec(prometheus.HistogramOpts{
-	Name:    "api_endpoints_response_time",
-	Help:    "API endpoints response time",
-	Buckets: prometheus.LinearBuckets(0, 20, 20),
-}, []string{"endpoint"})
-
-// APIResponseStatusCodes collects the information about api response status codes
-var APIResponseStatusCodes = promauto.NewCounterVec(prometheus.CounterOpts{
-	Name: "api_endpoints_status_codes",
-	Help: "API endpoints status codes",
-}, []string{"status_code"})
+var (
+	// APIRequests is a counter vector for requests to endpoints
+	APIRequests *prometheus.CounterVec = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
why do you need type here

Reply:
Oops, I did some test before initializing all the variables to `nil`, so I needed to define type, and later I returned to the original way to initialize but I forgot to remove the type initialization :-/ Obviously, you are right, the type definition is not needed at all